src/main/scala/mesosphere/marathon/api/v2/LeaderResource.scala

```diff
@@ -0,0 +1,27 @@
+package mesosphere.marathon.api.v2
+
+import javax.ws.rs.core.{ MediaType, Response }
+import javax.ws.rs.{ GET, DELETE, Path, Produces }
+
+import com.google.inject.Inject
+import mesosphere.chaos.http.HttpConf
+import mesosphere.marathon.api.RestResource
+import mesosphere.marathon.{ MarathonSchedulerService, MarathonConf }
+
+@Path("v2/leader")
+class LeaderResource @Inject() (
+    schedulerService: MarathonSchedulerService,
+    val config: MarathonConf with HttpConf)
+    extends RestResource {
+
```

Comment:
Could we also add a handler for the GET verb while we're in here?

- If there's a leader, respond with `{ "leader": "..." }`
- If there's no leader, respond with a `404`